fda-model/compute/FrequencyDamageMessage.cs

```diff
@@ -10,15 +10,12 @@ namespace compute
     public class FrequencyDamageMessage : HEC.MVVMFramework.Base.Interfaces.IMessage
     {
     private PairedData _frequencyDamage;
-        private string _damageCategory;
-        private string _assetCategory;
-        private int _impactAreaID;
     
     public string Message
         {
             get
             {
-                return $"This is the damage-frequency function for the impact area with ID {_impactAreaID}, damage category of {_damageCategory}, and asset category of {_assetCategory}";
+                return $"FrequencyDamage";
```

Comment:
The $ is not needed. I had to look that one up to see what the $ does:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
Is the message always going to be this or is this a placeholder for future work? There is no point in having a string property to return this. Just make it a public variable.

Reply:
let's talk more about that this afternoon.